presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Fix a bug where index joins would fail with the error ``driver should never block``.
+  Queries will now fail if the index is unable to load within the time specified by
+  the ``index-loader-timeout`` configuration property and the ``index_loader_timeout``
+  session property.
+* Fix a bug that could cause ``NullPointerException`` in spatial join with geometry
+  collections on the build side.
+* Improve performance for queries with ``FULL OUTER JOIN`` where join keys have the
+  ``COALESCE`` function applied.
+* Improve performance for ``UnnestOperator`` by using ``DictionaryBlock``.
+* Improve performance for ``PartitionedOutputOperator``. The new
```

Comment:
I think this is also, too low level (references class names that users don't necessarily know about). Instead I would say something like

```
Improve performance of repartitioning data between stages.  The optimization can be enabled by the ``optimized_repartitioning`` session property or the ``experimental.optimized-repartitioning`` configuration property.
```